lib/bolt/cli.rb

```diff
@@ -362,6 +379,32 @@ def run_plan(plan_name, plan_arguments, nodes, options)
       result.ok? ? 0 : 1
     end
 
+    def apply_manifest(code, targets, filename = nil)
+      require 'bolt/catalog'
```

Comment:
You don't seem to use anything from `Bolt::Catalog`. Is this just triggering `Bolt::PAL.load_puppet`?

Reply:
Whoops, I was using`Bolt::Catalog#parse_manifest` but changed it and left that in.